python/rpdk/typescript/codegen.py

```diff
@@ -22,11 +22,11 @@
 LOG = logging.getLogger(__name__)
 
 EXECUTABLE = "cfn"
-SUPPORT_LIB_NAME = "cfn-rpdk"
-SUPPORT_LIB_VERSION = "0.5.0"
+SUPPORT_LIB_NAME = (
+    "@amazon-web-services-cloudformation/cloudformation-cli-typescript-lib"
+)
+SUPPORT_LIB_VERSION = "^1.0.1"
```

Comment:
Nice. Now that the package will be published to NPM, the version here will only need to be changed for a major version.